### ISSUE TITLE
Matlab Version

### ISSUE THREAD

**Issue by usmanarif37** [uzh-rpg/rpg_svo#187]
Hi.
Thanks for sharing your work, really this is an act of kindness. I am new to C or ROS, so is Matlab version of this algorithm is also available..?

**Comment by alecGraves** [comment]
no. I don't think any other freely available version exists right now. You could make one though. I would be a good weekend project.
